sympy/stats/joint_rv_types.py

```diff
@@ -713,9 +713,9 @@ def GeneralizedMultivariateLogGamma(syms, delta, v, lamda, mu):
     ====
 
     If the GeneralizedMultivariateLogGamma is too long to type use,
-    `from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGamma as GMVLG`
+    ``from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGamma as GMVLG``
```

Comment:
Is this instruction really needed? Users can import it as any name as far as I am concerned.

Reply:
I wasn't convinced either. But at least it should be properly formatted...